plugins.d/tag.global.plugin

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+function tag.help {
+   # Select either of the two or both, depending on the functionality provided by the plugin
+   echo "  oc-cluster show-tags <ImageStreamName>"
+   echo "  oc-cluster tag-latest <ImageStreamName> <tag-name> - tags 'latest' ImageStream to <tag-name>"
+   echo "  oc-cluster tag-to-target <ImageStreamName> <source-tag> <target-project> <target-tag>"
+   echo "      Assumes that <source-tag> is something like 'RELEASE_1_1' and <target-tag> is 'latest'"
+   echo "      It first 'copies' the 'RELEASE_1_1' into <target-project> and call it 'latest'"
+   echo "      and then tags <target-project>:latest to 'RELEASE_1_1'"
+}
+
+#
+# Args:
+#  $1 ImageStreamName - the name of the application to tag
+#  $2 source-tag - the source tag to use
+#  $3 target-project - the name of the target project
+#  $4 target-tag - the name of the target tag in target-project
+#
+function tag-to-target {
+	[ $# -lt 4 ] && echo "ImageStream Name is required." && exit 1
```

Comment:
Why do you check for less than 4 parameters and then provide that error message?

Also, it would be cool to check if $1="-h or --help" and show the help for the command: 
https://github.com/openshift-evangelists/oc-cluster-wrapper/blob/master/oc-cluster#L591